src/profile/helpers/getServiceNames.ts

```diff
@@ -0,0 +1,13 @@
+import { ServiceConnectionQuery } from '../graphql/__generated__/ServiceConnectionQuery';
+
+export const getServiceTypes = (data?: ServiceConnectionQuery) => {
```

Comment:
I think it would be better if the export would have the same name as the module, or even better a default export! I would change this to `export default function getServiceNames(data?: ServiceConnectionQuery) { ... }`

Reply:
Changed export to default as suggested. I kept the original name (getServiceTypes) and renamed file to match that.